### PR TITLE
Filtering backend

### PR DESCRIPTION
**Pull request by MykytaMusaiev**
added some Filters fields and make it all works together

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 import {AppContext} from "./providers/context/appContext";
 import {useContext} from "react";
-import {I18nProvider, LOCALES} from "./providers/i18n";
+import {I18nProvider} from "./providers/i18n";
 
 const App = () => {
     const {state} = useContext(AppContext)
 
-    const httpLink = new HttpLink({uri: BASE_URL});
+    // const httpLink = new HttpLink({uri: `${location.origin}/graphql`});
```
Comment (TaHUoP) [inline client/src/App.js:11]: seems like you forgot to remove your debug code

Comment (anotherrandomcoder) [reply client/src/App.js:11]: will be removed in next task

Comment (TaHUoP) [reply client/src/App.js:11]: Then add todo

Comment (MykytaMusaiev) [reply client/src/App.js:11]: will be used before deployment to host

Comment (MykytaMusaiev) [reply client/src/App.js:11]: fixed in 609 task

**Review by TaHUoP — APPROVED** [review]

**Pull request merged**